specs/rest-api.md

```diff
@@ -699,9 +699,11 @@ In the source container case, the following dict must be passed:
 
     {
         "public": true,             # True or False
+        "name":   imagename,        # Name for new image
         "source": {
-            "type": "container",    # One of "container" or "snapshot"
-            "name": "abc"
+            "type":   "container",  # One of "container" or "snapshot"
+            "name":   "abc"
+            "remote": "dakara",     # remote where container is found
```

Comment:
The server code doesn't have the concept of "remotes".

I think it'd also be rather wasteful to transfer the whole container across server and then turn it into an image.

The better way of doing this I think is to have the client request the image be generated on the source server but with an argument that tells the remote server to serve the resulting image over a websocket.

Once the websocket is open, the server on which you want to store the image can then connect and grab the image. In such a mode, the image would never appear in the source LXD's image store.